cpp/src/gandiva/cache.cc

```diff
@@ -20,10 +20,10 @@
 
 namespace gandiva {
 
-static const int DEFAULT_CACHE_SIZE = 500;
+static const size_t DEFAULT_CACHE_SIZE = 128 * 1024 * 1024;  // 256 MiB
 
-int GetCapacity() {
-  int capacity;
+size_t GetCapacity() {
+  size_t capacity;
   const char* env_cache_size = std::getenv("GANDIVA_CACHE_SIZE");
   if (env_cache_size != nullptr) {
     capacity = std::atoi(env_cache_size);
```

Comment:
since it is calculating size it will be capped to int_max (2GB). Better to parse it as long.

Reply:
Done